bands/models.py

```diff
@@ -136,7 +136,7 @@ def w9_extension(self):
 
     @property
     def status(self):
-        return 'Yes <a href="{}">(view file)</a>'.format(self.w9_url) if self.w9_filename else ''
+        return self.w9_url if self.w9_url else ''
```

Comment:
This doesn't actually seem to work. I tried it real quick and it became falsey even though `w9_url` existed.